test/e2e/providers/aws/aws.go

```diff
@@ -325,14 +324,14 @@ func (a *awsProvider) GenerateMachineSet(withWindowsLabel bool, replicas int32)
 	if err != nil {
 		return nil, fmt.Errorf("unable to get subnet: %v", err)
 	}
-	machineSetName := "e2e-windows-machineset-"
+	machineSetName := "e2e-wmco"
 	matchLabels := map[string]string{
 		"machine.openshift.io/cluster-api-cluster": clusterName,
 	}
 
 	if withWindowsLabel {
-		matchLabels[windowsLabel] = "Windows"
-		machineSetName = machineSetName + "with-windows-label-"
+		matchLabels[clusterinfo.MachineOSIDLabel] = "Windows"
+		machineSetName = machineSetName + "w"
```

Comment:
Given that you are standardizing... The end result `e2e-wmcow` looks strange. I would rather we come up with a better naming convention.
```go
machineSetName := e2e-
if withWindowsLabel {
    machineSetName += "wmco"
} else {
    machineSetName += "nolabel"
}
```
Or something along those lines.

Reply:
As there is a limit of 9 characters I was limited in doing that.
I ended up using `e2e-wm` for machines meant to be configured by the Windows Machine controller, let me know what you think